services/processor/native/sanitizer/test/usecases/create_channel.go

```diff
@@ -0,0 +1,26 @@
+package usecases
+
+const CreateChannel = `package main
+
+import (
+	"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1"
+	"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/state"
+)
+
+var PUBLIC = sdk.Export(add)
+var SYSTEM = sdk.Export(_init)
+
+var COUNTER_KEY = []byte("count")
```

Comment:
please remove any non-pertinent code from these files, otherwise people will not understand why it's there

Reply:
Done.